deploy/kubernetes/overlays/local/ingress-nginx/baremetal/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
could you add a README explaining why there are 3 flavours under `ingress-nginx`/ and what the distinctions are? I can sort of guess by the name but i'd prefer to avoid the ambiguity.

I think you also mentioned that some of these files were copies of strongly inspired from other files? A link to those base files and the potential changes between base and these copies would be helpful. Maybe a hint in the source to remind to update these as well?